### ISSUE TITLE
Develop a guide that improves accessibility while development itself.

### ISSUE THREAD

**Issue by Dhruvi16** [creativecommons/cccatalog-frontend#618]
## Problem Description

A common accessibility myth is that accessibility is an expensive "added extra" to implement on a project. If however, we consider accessibility from the start and parallelly of a project, the cost of making most content accessible should be fairly minimal. So, to keep the development in favor of accessibility there should be a guide that tells the developer the importance of accessibility for this platform and how to maintain it. While it can certainly be helpful to run an automated testing tool on a service, it’s important that we don’t rely on them too heavily. No tool will be able to pick up every accessibility barrier on a website. So just because a tool hasn’t picked up any accessibility issues on a website, doesn’t mean those issues don’t exist.

## Solution Description

Create a guide that at first describe how important accessibility in terms of SEO, Mobile version, and also ease in development. This guide also layout rules of what to keep in mind while coding solutions. This guide can help us adapt better practices and those will be helpful to us in the long run.

**Comment by Dhruvi16** [comment]
I am willing to take this up.
Thank you!

**Comment by akmadian** [comment]
There are a number of guides elsewhere, and the (possible) introduction of pa11y (see #599), would help us maintain accessibility standards. Personally, I don't think this is necessary, but I'm interested in what @brenoferreira thinks.

**Comment by Dhruvi16** [comment]
Yeah! Makes sense. Let's see what @brenoferreira  thinks. Thank you for the response though!

**Comment by annatuma** [comment]
Thanks for the suggestion @Dhruvi16 but I agree with @akmadian here.

We are still working on finalizing our Design Library (in Figma), which will be reflected in Vocabulary. It will include examples on how to use components.

All components in our Design Library are designed through the lens of accessibility, and therefore this guide will end up existing in the Vocabulary Storybook.

Closing out this issue.

FYI @panchovm @brenoferreira
